SourceX/DiabloUI/selhero.cpp

```diff
@@ -370,4 +375,53 @@ BOOL UiSelHeroMultDialog(
 	selhero_isMultiPlayer = true;
 	return UiSelHeroDialog(fninfo, fncreate, fnstats, fnremove, dlgresult, name);
 }
+
+const char *selhero_GenerateName(std::uint8_t hero_class)
+{
+	// TODO: Replace these with lore-appropriate names. Currently they're generic
+	// fantasy names.
+	static const char *const kNames[3][10] = {
+		{
+		    "Arth",
+		    "Blulcorth",
+		    "Eadweard",
+		    "Eadwulf",
+		    "Ferthun",
+		    "Gurstudd",
+		    "Nehlem",
+		    "Tulvim",
+		    "Uriel",
+		    "Walmund",
+		},
+		{
+		    "Arleta",
+		    "Beata",
+		    "Emilia",
+		    "Galiana",
+		    "Kaliyah",
+		    "Margret",
+		    "Pascaline",
+		    "Todra",
+		    "Uta",
+		    "Veho",
+		},
+		{
+		    "Ellinon",

```

Comment:
Here is a better list of mage names:
```
Jazreth
Drognan
Armin
Fauztin
Jere
Kazzulk
Ranslor
Sarnakyle
Valthek
Horazon
```
Jazreth is the official name of the mage, all other names are of mages from the same guild, Drognan appears in D2 as a trader. I picked the 10 most likely mages to have been involved in D1 from various D lore, but some of them are probably anachronistic but no one says that there can't have been two people with the same first name in history :)

Reply:
Done